plugins/filter/record_transformer.md

```diff
@@ -59,6 +59,40 @@ into
 {"total":100, "count":10, "avg":"10"}
 ```
 
+This filter can be also used to remove fields through the `remove_keys`
+parameter (including nested keys, by means of `jsonpath` syntax)
```

Comment:
> (including nested keys, by means of `jsonpath` syntax)

Remove this sentence. We don't say this syntax is `jsonpath` and `remove_keys` parameter section has a link to record accessor's syntax page.